content/en/docs/tasks/tools/install-kubectl.md

```diff
@@ -117,33 +145,48 @@ kubectl version --client
 
 ### Install kubectl binary with curl on macOS
 
-1. Download the latest release:
+1. Download the latest stable release:
 
-   ```bash
-   curl -LO "https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/darwin/amd64/kubectl"
-   ```
+    ```bash
+    curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/darwin/amd64/kubectl"
+    ```
+    {{< note >}}
+To download a specific version, replace the `$(curl -L -s https://dl.k8s.io/release/stable.txt)` portion of the command with the specific version.
```

Comment:
The indentation looks off; is that deliberate?

Reply:
The text can pushed over -- it's actually flush left to be consistent with the next line. When you use ``` in a `note` it needs to be flush left or it doesn't render correctly.

![Screen Shot 2020-12-16 at 3 05 47 PM](https://user-images.githubusercontent.com/4453979/102417395-81126000-3ff3-11eb-9b8f-35e19ec87be9.png)

Reply:
I'll update what can be indented correctly, and only keep the part that needs to be flush left to work flush left.

Reply:
See https://github.com/kubernetes/website/pull/25259#discussion_r544745665

With the right indentation overall, you should not need to move the note shortcode to be flush-left.

Reply:
If that doesn't work, _or_ if the examples aren't clear, let's track that as an issue and then see about scheduling a fix.